contracts/eden/src/actions/induct.cpp

```diff
@@ -70,6 +73,21 @@ namespace eden
       inductions.endorse(induction, account, induction_data_hash);
    }
 
+   void eden::inductpayfee(eosio::name payer, uint64_t id, const eosio::asset& quantity)
+   {
+      eosio::require_auth(payer);
+
+      globals globals{get_self()};
+      inductions inductions{get_self()};
+      accounts accounts{get_self()};
+
+      const auto& induction = inductions.get_induction(id);
+      eosio::check(payer == induction.invitee(), "only inductee may pay fee");
+      eosio::check(quantity == globals.get().minimum_donation, "incorrect fee");
```

Comment:
I believe here we can verify for greater or equal than minimum donation fee (for the cases where the new member wants to donate a bigger amount)

Reply:
A separate donation action may help reduce mistakes. The UI could make it clear that donations aren't usable for induction or election fees.